modules/caddytls/connpolicy.go

```diff
@@ -244,14 +244,18 @@ type ClientAuthentication struct {
 	// which are not in this list will be rejected.
 	TrustedLeafCerts []string `json:"trusted_leaf_certs,omitempty"`
 
+	// requires a client certificate, but will not verify it
+	ClientAuthType string `json:"client_auth_type,omitempty"`
```

Comment:
```suggestion
	Mode string `json:"mode,omitempty"`
```

Since we're already in the ClientAuthentication struct, the fact that it's Client Auth is obvious, and doesn't need to be repeated, I think. I also think of this as an operational mode than a type. Tbh I'm not a huge fan of the std lib's client auth configuration...